sites/postcss-preset-env/blog/postcss-bundler.md

```diff
@@ -0,0 +1,135 @@
+---
+title: PostCSS Bundler
+description: A standards compliant reference CSS bundler
+date: 2023-09-15
+---
+
+[PostCSS Bundler] is a standards compliant reference CSS bundler.
+
+It is built around `@import` statements, similarly to how [esbuild](https://esbuild.github.io), [postcss-import](https://www.npmjs.com/package/postcss-import) or [lightningcss](https://lightningcss.dev) bundle CSS.
+
+```css
+@import url('variables.css');
+@import url('typography.css');
+@import url('components/not-a-card.css');
+```
+
+
+## Why this is matters now
+
+Previously bundling CSS based on `@import` was relatively simple.
+
+There were always a few features (`@namespace`, `@charset`, ...) that weren't or couldn't be implemented correctly in bundlers but these also happened to be obscure and rarely used features.
+
+CSS Bundlers had the luxury of getting by with naive implementations.  
+Even with duplicate and cyclical imports it was largely invisible if a bundler incorrectly only included the first or the last import.
+
+Today however this is no longer true.
+
+We now have cascade layers, support conditions and soon even scoping in `@import` statements.  
+With the addition of these features the source order is more important than ever and bundlers need to be updated so that you can use all of the features in CSS.
```

Comment:
```suggestion
With the addition of these features, the source order is more important than ever and bundlers need to be updated so that you can use all of the features in CSS.
```